resources/sql/create_tables.sql

```diff
@@ -19,7 +19,7 @@ CREATE TABLE IF NOT EXISTS Pessoa (
     dataNascimento DATE NOT NULL,
     celular BIGINT,
     telefone BIGINT,
-    email VARCHAR(255) UNIQUE,
+    email VARCHAR(255) NOT NULL UNIQUE,
```

Comment:
Você não tinha comentado sobre o email poder ser null, pq colocou que é not null?

Reply:
Pois como você mesma pode ver o e-mail está declarado como um dado único, ou seja não pode existir duplicidade no banco, sendo assim se existir um e-mail vazio ou igual a Null já cadastrado, irá retornar um erro. Por isso coloque que não pode estar vazio, pq mesmo se der um erro é porque o e-mail já está cadastrado, e não porque está retornando um e-mail vazio existente.